src/components/topNav/primaryTopNav/PrimaryTopNav.module.css

```diff
@@ -4,6 +4,7 @@
   --conduction-primary-top-nav-background-color: var(--skeleton-color-secondary-3);
   --conduction-primary-top-nav-background-color-hover: var(--skeleton-color-secondary-5);
   --conduction-primary-top-nav-dropdown-border-radius: var(--skeleton-border-radius-md);
+  --conduction-primary-top-nav-background-color-active: hsla(0, 0%, 100%, 0.3);
```

Comment:
Even though `hsla` works fine, we're using `rgba` throughout the codebase, therefore I suggest changing it to `rgba`.

Which would result in `rgba(255, 255, 255, 0.3)`. I'd opt to make it `0.2` as well.